scripts/delete-resource-quota.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+oc delete -f ./test-target/resource-quota.yaml --namespace "$TNF_EXAMPLE_CNF_NAMES" --ignore-not-found
```

Comment:
New line is missing.

Reply:
Use descriptive flags in scripts: --filename instead of -f

Reply:
Consider to format:
```
oc delete \
    --filename ./test-target/resource-quota.yaml \
    --namespace "$TNF_EXAMPLE_CNF_NAMES" \
    --ignore-not-found
```

Reply:
Newline added and descriptive flag used.  I think I'm good with the single line for now.  We can readjust later if the lines get any longer/crazier.